### PR TITLE
Docblocks

### PR DESCRIPTION
**Pull request by elinw**

### CODE REVIEW

File: libraries/joomla/document/document.php [hunk 1]
```diff
@@ -237,7 +237,7 @@ class JDocument extends JObject
 	 * if it doesn't already exist.
 	 *
 	 * @param   string  $type       The document type to instantiate
```
Comment (eddieajau) [inline libraries/joomla/document/document.php:239]: Need to align the description of the type param.

File: libraries/joomla/document/document.php [hunk 2]
```diff
@@ -448,8 +450,8 @@ class JDocument extends JObject
 	/**
 	 * Adds a script to the page
 	 *
-	 * @param   string  $content	Script
-	 * @param   string  $type	Scripting mime (defaults to 'text/javascript')
+	 * @param   string  $content Script
```
Comment (eddieajau) [inline libraries/joomla/document/document.php:454]: Check there are 2 spaces after $content

File: libraries/joomla/document/document.php [hunk 3]
```diff
@@ -531,9 +539,11 @@ class JDocument extends JObject
 	/**
 	 * Sets the global document language declaration. Default is English (en-gb).
 	 *
-	 * @param   string	$lang
+	 * @param   string   $lang  The language to be set
```
Comment (eddieajau) [inline libraries/joomla/document/document.php:543]: Check there are 2spaces after string

File: libraries/joomla/document/document.php [hunk 4]
```diff
@@ -598,9 +617,11 @@ class JDocument extends JObject
 	/**
 	 * Sets the base URI of the document
 	 *
-	 * @param  string  $base
+	 * @param   string   $base  The base URI to be set
```
Comment (eddieajau) [inline libraries/joomla/document/document.php:621]: Check there 2spacesafter string

File: libraries/joomla/document/document.php [hunk 5]
```diff
@@ -804,7 +855,7 @@ class JDocument extends JObject
 	 *
 	 * @param   string  $type  The renderer type
 	 *
-	 * @return  mixed  Object or null if class does not exist
+	 * @return  mixed   Object or null if class does not exist
```
Comment (eddieajau) [inline libraries/joomla/document/document.php:859]: Check there are only 2spaces after mixed.

File: libraries/joomla/filesystem/file.php [hunk 6]
```diff
@@ -512,7 +515,7 @@ class JFile
 	/**
 	 * Wrapper for the standard file_exists function
 	 *
-	 * @param   string  $file  File path
+	 * @param   string   $file  File path
```
Comment (eddieajau) [inline libraries/joomla/filesystem/file.php:518]: Check for 2spaces after string

**Comment by elinw** [comment]
There are supposed to be a minimum of two spaces as I understand the rule.

**Pull request merged**